packages/jaeger-ui/src/components/SearchTracePage/url.tsx

```diff
@@ -20,6 +20,7 @@ import prefixUrl from '../../utils/prefix-url';
 import { MAX_LENGTH } from '../DeepDependencies/Graph/DdgNodeContent/constants';
 
 import { SearchQuery } from '../../types/search';
+import parseQuery from '../../utils/parseQuery.test';
```

Comment:
why are we importing test module into prod module?

Reply:
I think that way we won't have to worry about its code coverage. Changing it back to normal .ts file